app/models/upload.rb

```diff
@@ -6,6 +6,7 @@ class Upload < ApplicationRecord
   belongs_to :stream, touch: true
   has_one :organization, through: :stream
   has_many :marc_records, dependent: :delete_all
+  belongs_to :user, optional: true
```

Comment:
Should this belong to any `User` who touches it?

Reply:
If the files change, I guess? The goal with #179 is to give us some clues if/when users report upload trouble or something.